internal/commands/app/diff.go

```diff
@@ -60,13 +71,35 @@ func (cmd *CommandDiff) Flags() []flags.Flag {
 			},
 		},
 		flags.BoolFlag{
-			Value: &cmd.inputs.IncludeDependencies,
+			Value: &cmd.inputs.IncludeNodeModules,
+			Meta: flags.Meta{
+				Name: flagIncludeNodeModules,
+				Usage: flags.Usage{
+					Description: "Include Realm app dependencies in the diff from a node_modules archive",
+					Note:        "The allowed formats are as a directory or compressed into a .zip, .tar, .tar.gz, or .tgz file",
+				},
+			},
+		},
+		flags.BoolFlag{
+			Value: &cmd.inputs.IncludePackageJSON,
+			Meta: flags.Meta{
+				Name: "include-package-json",
+				Usage: flags.Usage{
+					Description: "Include Realm app dependencies in the diff from a package.json file",
+				},
+			},
+		},
+		// TODO(REALMC-10088): Remove this flag in realm-cli 3.x
+		flags.BoolFlag{
+			Value: &cmd.inputs.IncludeNodeModules,
```

Comment:
this should still be `&cmd.inputs.IncludeDependencies` (I think this got reverted somehow?)